File: tests/benchmarks/cases/fib_recursive/java/benchmark_case.java
final class FibRecursiveCase {
    static final String NAME = "fib_recursive";
    static final String PASS_BANNER = "BENCH_FIB_RECURSIVE_PASS";

    private FibRecursiveCase() {}

    static long run(int scale) {
        return BenchmarkSupport.fibRecursive(scale);
    }
}

File: tests/benchmarks/cases/fib_recursive/c/benchmark_case.c
#include "benchmark_case.h"
#include "benchmark_support.h"

static ZrBenchInt zr_bench_case_fib_recursive_run(int scale) {
    return zr_bench_run_fib_recursive(scale);
}

const ZrBenchCaseDescriptor zr_bench_case_descriptor_fib_recursive = {
        "fib_recursive",
        "BENCH_FIB_RECURSIVE_PASS",
        zr_bench_case_fib_recursive_run
};
